### ISSUE TITLE
Database Deletion Fails

### ISSUE THREAD

**Issue by eberlep** [fi-ts/postgreslet#129]
```
postgreslet-8c64457cc-8b54t postgreslet 2021-02-22T10:54:10.311Z        ERROR   controller-runtime.manager.controller.postgres  Reconciler error        {"reconciler group": "database.fits.cloud", "reconciler kind": "Postgres", "name": "741e4e87-c3e4-48f8-b175-a29b0f3b08a7", "namesp
ace": "metal-extension-cloud", "error": "unable to create or update corresponding CRD ClusterwideNetworkPolicy: &{%!W(string=unable to deploy CRD ClusterwideNetworkPolicy: clusterwidenetworkpolicies.metal-stack.io \"b621eb9948884911-6eef363b7abe4b729e07\" already exists) %!W(*error
s.StatusError=&{{{ } {   <nil>} Failure clusterwidenetworkpolicies.metal-stack.io \"b621eb9948884911-6eef363b7abe4b729e07\" already exists AlreadyExists 0xc000833aa0 409}})}"}                                                                                                           
postgreslet-8c64457cc-8b54t postgreslet sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem                                                                                                                                                          
postgreslet-8c64457cc-8b54t postgreslet         /go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.2/pkg/internal/controller/controller.go:252                                                                                                                                               
postgreslet-8c64457cc-8b54t postgreslet sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1.2                                                                                                                                                                
postgreslet-8c64457cc-8b54t postgreslet         /go/pkg/mod/sigs.k8s.io/controller-runtime@v0.8.2/pkg/internal/controller/controller.go:215                                                                                                                                               
postgreslet-8c64457cc-8b54t postgreslet k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext.func1                                                                                                                                                                                    
postgreslet-8c64457cc-8b54t postgreslet         /go/pkg/mod/k8s.io/apimachinery@v0.20.4/pkg/util/wait/wait.go:185                                                                                                                                                                         
postgreslet-8c64457cc-8b54t postgreslet k8s.io/apimachinery/pkg/util/wait.BackoffUntil.func1                                                                                                                                                                                              
postgreslet-8c64457cc-8b54t postgreslet         /go/pkg/mod/k8s.io/apimachinery@v0.20.4/pkg/util/wait/wait.go:155                                                                                                                                                                         
postgreslet-8c64457cc-8b54t postgreslet k8s.io/apimachinery/pkg/util/wait.BackoffUntil                                                                                                                                                                                                    
postgreslet-8c64457cc-8b54t postgreslet         /go/pkg/mod/k8s.io/apimachinery@v0.20.4/pkg/util/wait/wait.go:156                                                                                                                                                                         
postgreslet-8c64457cc-8b54t postgreslet k8s.io/apimachinery/pkg/util/wait.JitterUntil                                                                                                                                                                                                     
postgreslet-8c64457cc-8b54t postgreslet         /go/pkg/mod/k8s.io/apimachinery@v0.20.4/pkg/util/wait/wait.go:133                                                                                                                                                                         
postgreslet-8c64457cc-8b54t postgreslet k8s.io/apimachinery/pkg/util/wait.JitterUntilWithContext                                                                                                                                                                                          
postgreslet-8c64457cc-8b54t postgreslet         /go/pkg/mod/k8s.io/apimachinery@v0.20.4/pkg/util/wait/wait.go:185                                                                                                                                                                         
postgreslet-8c64457cc-8b54t postgreslet k8s.io/apimachinery/pkg/util/wait.UntilWithContext                                                                                                                                                                                                
postgreslet-8c64457cc-8b54t postgreslet         /go/pkg/mod/k8s.io/apimachinery@v0.20.4/pkg/util/wait/wait.go:99  
```

After the controller runtime update, deletion of (newly created) databases seems to fail.

**Comment by eberlep** [comment]
Occured in #124

**Comment by majst01** [comment]
Tried to address this with: https://github.com/fi-ts/postgreslet/pull/124/commits/f5f632ff25d45a51a12bbaf68d908de9b91e5e25

**Comment by eberlep** [comment]
Looks like the right spot, but why did it happen in the first place? Or why now and never before?

**Comment by majst01** [comment]
fixed with #124
